packages/react-components/src/components/CheckBox.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useRuleEngine, useFormIntl } from '@aemforms/af-react-renderer';
+import {State, FieldJson} from '@aemforms/af-core';
+import '../../dist/theme.css';
```

Comment:
remove theme.css line.
import theme.css in your test application

Reply:
Removed!